config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'sessions#new'
```

Comment:
Could you move the root path to the bottom of the file? Otherwise when Rails automatically adds generated routes to this file, the root path will end up being hard to find amidst the rest of the routes.